.github/workflows/ci-linux-on-main-pr.yml

```diff
@@ -24,18 +22,20 @@ jobs:
       - name: "Installing dependencies"
         run: |
           sudo apt-get update -y
-          sudo apt-get install -y
+          sudo apt-get install -y 
           sudo apt-get remove -y llvm-12
           sudo apt-get install -y curl pkg-config findutils  wget
-          sudo apt install -y clang-11 cmake clang-format-11 clang-tidy-11
-          sudo apt-get install -y llvm-11 lldb-11 llvm-11-dev libllvm11 llvm-11-runtime opt
-          sudo apt install -y python3 python3-pip
+          sudo apt install -y clang-11 cmake clang-format-11 clang-tidy-11 
+          sudo apt-get install -y llvm-11 lldb-11 llvm-11-dev libllvm11 llvm-11-runtime opt 
+          sudo apt-get install -y nodejs npm
+          sudo apt install -y python3 python3-pip 
           sudo apt-get remove -y llvm-10
           sudo ln -s /usr/lib/llvm-11/bin/opt /usr/bin/opt
-          sudo update-alternatives --install /usr/bin/python python /usr/bin/python3 0
-          dotnet tool install --global Microsoft.Quantum.IQSharp --version 0.20.2110171573
+          sudo update-alternatives --install /usr/bin/python python /usr/bin/python3 0       
```

Comment:
Here it seems some additional whitespace was added to the end of the line that probably should be removed (and the line below too).

Reply:
I am surprised that my formatter does not remove this. Will just double check that it is actually correctly set up.

Reply:
OK, so I understand whats going on here: Since `run:` is a multiline text block, it does not change the spaces as they could, so it will need to be done manually :'(